cli/main.py

```diff
@@ -42,13 +42,23 @@ def cli(
         else Path(os.path.expanduser(config))
     )
 
-    ctx.obj = CliContext(confman=ConfigurationManager(conf_path, profile, verbose))
+    auth_api_url_env_var = get_auth_api_url_env_var()
+    auth_api_url = URL(auth_api_url_env_var) if auth_api_url_env_var else None
+
```

Comment:
```python
auth_api_url_env_var = os.environ.get('AUTH_API_URL')
auth_api_url = URL(auth_api_url_env_var) if auth_api_url_env_var else None
```
No need in `get_auth_api_url_env_var()`, `get` returns None if the key wasn't found.